### ISSUE TITLE
\r error

### ISSUE THREAD

**Issue by plinkboom** [FezVrasta/ark-server-tools#717]
I made a cluster server which includes the center, scorched earth, and the island. If I let the instance.cfg create a folder for the save file eg. ark_AltSaveDirectoryName="/SavedArks/TheCenter/ I get problems accessing that folder so I can't retrieve the saved files. I can see the folder via filezilla using root but I can't access it. But If I create a folder for the saved files and point to it I then get permission denied errors.. I am also getting \r command not found errors on blank lines on my config. Am I missing something on my server for ark server tools? Thank you. It's a dedicated ubuntu machine 16.04. Any help would really be appreciated. The server runs  under the user steam.

steam@server:~$ arkmanager start @all
Running command 'start' for instance 'TheCenter'
/etc/arkmanager/instances/TheCenter.cfg: line 3: $'\r': command not found
/etc/arkmanager/instances/TheCenter.cfg: line 7: $'\r': command not found
/etc/arkmanager/instances/TheCenter.cfg: line 25: $'\r': command not found
/etc/arkmanager/instances/TheCenter.cfg: line 29: $'\r': command not found
/etc/arkmanager/instances/TheCenter.cfg: line 32: $'\r': command not found
mkdir: cannot create directory ‘/home/steam/ARK/ShooterGame/Saved///SavedArks/TheCenter/\r’: Permission denied
[  ERROR  ]     The ARK SavedArks directory is not writable, and saveworld will fail
The server is now running, and should be up within 10 minutes
Running command 'start' for instance 'ScorchedEarth'
/etc/arkmanager/instances/ScorchedEarth.cfg: line 3: $'\r': command not found
/etc/arkmanager/instances/ScorchedEarth.cfg: line 7: $'\r': command not found
/etc/arkmanager/instances/ScorchedEarth.cfg: line 25: $'\r': command not found
/etc/arkmanager/instances/ScorchedEarth.cfg: line 29: $'\r': command not found
/etc/arkmanager/instances/ScorchedEarth.cfg: line 32: $'\r': command not found
mkdir: cannot create directory ‘/home/steam/ARK/ShooterGame/Saved//SavedArks/ScorchedEarth/\r’: Permission denied
[  ERROR  ]     The ARK SavedArks directory is not writable, and saveworld will fail
The server is now running, and should be up within 10 minutes
Running command 'start' for instance 'TheIsland'
/etc/arkmanager/instances/TheIsland.cfg: line 3: $'\r': command not found
/etc/arkmanager/instances/TheIsland.cfg: line 7: $'\r': command not found
/etc/arkmanager/instances/TheIsland.cfg: line 25: $'\r': command not found
/etc/arkmanager/instances/TheIsland.cfg: line 26: $'\r': command not found
/etc/arkmanager/instances/TheIsland.cfg: line 30: $'\r': command not found
/etc/arkmanager/instances/TheIsland.cfg: line 33: $'\r': command not found
mkdir: cannot create directory ‘/home/steam/ARK/ShooterGame/Saved//SavedArks/TheIsland/\r’: Permission denied
[  ERROR  ]     The ARK SavedArks directory is not writable, and saveworld will fail
The server is now running, and should be up within 10 minutes

**Comment by klightspeed** [comment]
You will need to convert the Windows CR/LF line endings in the files to Unix LF line endings.

You can do this by running:
```
sed -i 's/\r$//g' /etc/arkmanager/instances/*.cfg
```

Also note that the ARK server will tighten the security on its config and save files to ensure that only the user it is running under (`steam` in this case) can access those files.  Note that `root` can always access these files, as it can bypass file permission checks.

**Comment by plinkboom** [comment]
Thank you klightspeed. I'll test it out! =)

**Comment by plinkboom** [comment]
klightspeed it worked. I am not getting any \r command not found errors any more. Thank you for the solution. I'm still getting errors accessing the saved files while using root through filezilla though. Any idea what might be causing this? Thank you.

Status:	Retrieving directory listing of "/home/steam/ARK/ShooterGame/Saved/SavedArks/TheIsland/Jan 15 21:30"...
Command:	cd "/home/steam/ARK/ShooterGame/Saved/SavedArks/TheIsland"
Response:	New directory is: "/home/steam/ARK/ShooterGame/Saved/SavedArks/TheIsland"
Command:	cd "Jan 15 21:30"
Error:	Directory /home/steam/ARK/ShooterGame/Saved/SavedArks/TheIsland/Jan 15 21:30: no such file or directory
Error:	Failed to retrieve directory listing
Status:	Retrieving directory listing of "/home/steam/ARK/ShooterGame/Saved/SavedArks/TheCenter/Jan 15 21:30"...
Command:	cd "/home/steam/ARK/ShooterGame/Saved/SavedArks/TheCenter/Jan 15 21:30"
Error:	Directory /home/steam/ARK/ShooterGame/Saved/SavedArks/TheCenter/Jan 15 21:30: no such file or directory
Error:	Failed to retrieve directory listing
Status:	Retrieving directory listing of "/home/steam/ARK/ShooterGame/Saved/SavedArks/ScorchedEarth/Jan 15 21:30"...
Command:	cd "/home/steam/ARK/ShooterGame/Saved/SavedArks/ScorchedEarth/Jan 15 21:30"
Error:	Directory /home/steam/ARK/ShooterGame/Saved/SavedArks/ScorchedEarth/Jan 15 21:30: no such file or directory
Error:	Failed to retrieve directory listing

**Comment by klightspeed** [comment]
You are trying to list the `Jan 15 21:30` directory within `/home/steam/ARK/ShooterGame/Saved/SavedArks/TheIsland`, `/home/steam/ARK/ShooterGame/Saved/SavedArks/TheCenter` and `/home/steam/ARK/ShooterGame/Saved/SavedArks/ScorchedEarth`

**Comment by plinkboom** [comment]
Yeah, Isn't that the save files for my ark server? I wanted to download it as backup.

**Comment by klightspeed** [comment]
The name will be `TheIsland.ark` or `TheCenter.ark` or `ScorchedEarth.ark`
